### ISSUE TITLE
Cycling of refresh tokens

### ISSUE THREAD

**Issue by nghamilton** [ConsumerDataStandardsAustralia/standards-maintenance#50]
The Standards indicate that the expiry of the currently active refresh token ends a sharing arrangement. However, there is a provision in the standards to cycle refresh tokens, which indicates that the 'currently active' refresh token could be made inactive and the sharing arrangement would still continue. 

Suggest rewording the first statement to clarify:

`The revocation or expiration of the currently active refresh token should be understood to effectively revoke or expire the sharing arrangement as a whole.`

**Comment by CDR-API-Stream** [comment]
It is unclear what is being suggested here.  The standards already contain the sentence suggested.

**Comment by nghamilton** [comment]
The current wording is the one above (i.e `The revocation or expiration of the currently active refresh token should be understood to effectively revoke or expire the sharing arrangement as a whole.`).

I suggest it should be reworded to something like this:

`The revocation or expiration of all currently active refresh tokens associated with a sharing agreement should be understood to effectively revoke or expire the sharing arrangement as a whole.`

**Comment by CDR-API-Stream** [comment]
This was added to 1.1.0
